pallets/did/src/benchmarking.rs

```diff
@@ -759,6 +766,88 @@ benchmarks! {
 	verify {
 		assert!(!Did::<T>::get(&did_subject).unwrap().key_agreement_keys.contains(&key_agreement_key_id));
 	}
+
+	signature_verification_sr {
```

Comment:
C: I would add the final `25519`, just in case.

Reply:
6b058202dbd346633063c762ed0ef0f90a2898f1